test/ModifyLiquidity.t.sol

```diff
@@ -263,45 +263,45 @@ contract ModifyLiquidityTest is Test, Logger, Deployers, JavascriptFfi, Fuzzers,
             IPoolManager.ModifyLiquidityParams({tickLower: -120, tickUpper: 120, liquidityDelta: 2e18, salt: SALT});
 
         // Get the uninitialized positions and assert they have no liquidity.
-        Position.Info memory positionSalt = manager.getPosition(
+        (uint128 liquiditySalt,,) = manager.getPositionInfo(
             simplePoolId, address(modifyLiquidityRouter), LIQ_PARAM_SALT.tickLower, LIQ_PARAM_SALT.tickUpper, SALT
         );
 
-        Position.Info memory positionSalt2 = manager.getPosition(
+        (uint128 liquiditySalt2,,) = manager.getPositionInfo(
             simplePoolId, address(modifyLiquidityRouter2), LIQ_PARAM_SALT_2.tickLower, LIQ_PARAM_SALT_2.tickUpper, SALT
         );
 
-        assertEq(positionSalt.liquidity, 0);
-        assertEq(positionSalt2.liquidity, 0);
+        assertEq(liquiditySalt, 0);
+        assertEq(liquiditySalt2, 0);
 
         // Modify the liquidity with the salt with the first router.
         modifyLiquidityRouter.modifyLiquidity(simpleKey, LIQ_PARAM_SALT, ZERO_BYTES);
 
-        Position.Info memory updatedPositionSalt = manager.getPosition(
+        (uint128 updatedLiquiditySalt,,) = manager.getPositionInfo(
             simplePoolId, address(modifyLiquidityRouter), LIQ_PARAM_SALT.tickLower, LIQ_PARAM_SALT.tickUpper, SALT
         );
 
-        Position.Info memory updatedPositionSalt2 = manager.getPosition(
-            simplePoolId, address(modifyLiquidityRouter2), LIQ_PARAM_SALT.tickLower, LIQ_PARAM_SALT.tickUpper, SALT
+        (uint128 updatedLiquiditySalt2,,) = manager.getPositionInfo(
+            simplePoolId, address(modifyLiquidityRouter2), LIQ_PARAM_SALT_2.tickLower, LIQ_PARAM_SALT_2.tickUpper, SALT
```

Comment:
was this just an incorrect test before? why was it passing?

Reply:
they are actually the same params so I do think it was a mistake from before but the test itself was still "correct"